manifests/config.pp

```diff
@@ -151,17 +153,21 @@
   # Configuration logic ends, resources begin:
 
   file { "${jira::webappdir}/bin/user.sh":
-    content => epp('jira/user.sh.epp'),
+    content => epp("${module_name}/user.sh.epp"),
```

Comment:
could you provide this change (and the other replacements of the hardcoded `jira` string) as a dedicated PR? That reduces the diff and makes the PR easier to review

Reply:
Ok, I'm fine with both strategies, so I suggest @oranenj an @bastelfreak agree on one or the other?

Reply:
Pick whichever approach you feel is best, I don't mind either way.